### ISSUE TITLE
Experiments with cifar-10

### ISSUE THREAD

**Issue by a7b23** [facebookresearch/ImageNet-Adversarial-Training#4]
Hi, as per the Madry's baseline the robust model has a PGD accuracy of 47% for cifar-10 dataset. Does the feature denoising technique improve the robustness of the model for the cifar-10 too?
Do you have any experimental result for cifar-10 ?

**Comment by ppwwyyxx** [comment]
Thanks for the interest in our work.
We do not have much experience or interest in Cifar10 since the observations on this small dataset often do not transfer to practice. From our very limited number of experiments, feature denoising also improves robustness on cifar10.

**Comment by cihangxie** [comment]
For your reference, our LIMITED CIFAR-10 results are posted here:
 
<table>
<thead>
<tr>
<th align="left" rowspan=2>Model </th>
<th align="center" colspan=3>accuracy (%)</th>
</tr>
<tr>
<th align="center">clean images</th>
<th align="center">20-step PGD</th>
<th align="center">100-step PGD</th>
</tr>
</thead>


<tbody>
<tr>
<td align="left"><summary>Simple</summary></td>
<td align="center">83.8</td>
<td align="center">44.1</td>
<td align="center">43.5</td>
</tr>

<tr>
<td align="left"><summary>Simple Denoise</summary></td>
<td align="center">84.3</td>
<td align="center">45.2</td>
<td align="center">44.6</td>
</tr>

<tr>
<td align="left"><summary>Wide</summary></td>
<td align="center">86.2</td>
<td align="center">46.7</td>
<td align="center">46.4</td>
</tr>

<tr>
<td align="left"><summary>Wide Denoise</summary></td>
<td align="center">86.3</td>
<td align="center">47.7</td>
<td align="center">47.1</td>
</tr>
</tbody>
</table>

In summary, from Simple to Wide (10x parameters),  it gives you ~3% improvement. Adding 4 denoising layers (only a few more parameters), it gives you ~1% improvement. But as yuxin commented, we do not have much experience or interest in Cifar10, these experiment results are done personally only for a result verification purpose.
